packages/lesswrong/lib/collections/posts/views.js

```diff
@@ -184,10 +184,11 @@ ensureIndex(Posts,
   }
 );
 
+const frontpageSelector = {frontpageDate: {$gte: new Date(0)}}
+if (getSetting('EAForum')) frontpageSelector.meta = {$ne: true}
```

Comment:
It's a `const`! Just because javascript allows you to lie to yourself, doesn't mean you should :P